### PR TITLE
Turn JsonOutput into a pair of free functions.

### PR DESCRIPTION
**Pull request by pphaneuf**

### CODE REVIEW

File: cpp/server/json_output.cc [hunk 1]
```diff
@@ -68,12 +68,8 @@ string LogRequest(evhttp_request* req, int http_status, int resp_body_length) {
...
 
 
-JsonOutput::JsonOutput(libevent::Base* base) : base_(CHECK_NOTNULL(base)) {
-}
-
-
-void JsonOutput::SendJsonReply(evhttp_request* req, int http_status,
-                               const JsonObject& json) {
+void SendJsonReply(libevent::Base* base, evhttp_request* req, int http_status,
+                   const JsonObject& json) {
```
Comment (AlCutter) [inline cpp/server/json_output.cc]: `CHECK_NOTNULL(base)` `CHECK_NOTNULL(req)`
?

File: cpp/server/json_output.cc [hunk 2]
```diff
@@ -95,20 +91,20 @@ void JsonOutput::SendJsonReply(evhttp_request* req, int http_status,
...
   } else {
     send_reply();
   }
 }
 
 
-void JsonOutput::SendError(evhttp_request* req, int http_status,
-                           const string& error_msg) {
+void SendJsonError(libevent::Base* base, evhttp_request* req, int http_status,
+                   const string& error_msg) {
```
Comment (AlCutter) [inline cpp/server/json_output.cc]: same here

File: cpp/server/json_output.cc (hunk 1 above)
Comment (pphaneuf) [inline cpp/server/json_output.cc]: Done.

File: cpp/server/json_output.cc (hunk 2 above)
Comment (pphaneuf) [inline cpp/server/json_output.cc]: They're passed straight through to the other function, so I'll let it do it...

Comment (AlCutter) [inline cpp/server/json_output.cc]: ah yes, of course they are!

**Pull request merged**